### DISCUSSION TITLE
Cloudinary Typescript

### DISCUSSION THREAD

**Discussion by WalterRestifo** [neuefische/questions#414] [Web] [answered]
Hello, I´m trying to use cloudinary to upload a profile picture to a cloud, take the src url and write it in my mongodb. I want to use next-cloudinary "a developers first-class support for using Cloudinary in Next.js applications" (quote from the next-cloudinary webpage).
I installed everything like written in the get started docs, but when I try to import { CldImage } from "next-cloudinary" I have this problem: 
Could not find a declaration file for module 'next-cloudinary'. '/Users/walterrestifo/Desktop/web-projects/cap-pro-ts/node_modules/next-cloudinary/dist/next-cloudinary.umd.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/next-cloudinary` if it exists or add a new declaration (.d.ts) file containing `declare module 'next-cloudinary';`ts(7016)

I wrote npm i --save-dev @types/next-cloudinary in my terminal, but I have errors like:
E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@types%2fnext-cloudinary - Not found
npm ERR! 404 
npm ERR! 404  '@types/next-cloudinary@*' is not in this registry.

What should I do?

**Comment by WalterRestifo** [comment, marked as answer]
Hi, I had to make a new file next-cloudinary.d.ts and code:
declare module "next-cloudinary";
in it. Now all components of next cloudinary have an any type.

**Reply by onebarloop** [reply]
Thanks Walter, this is important information!
